ghost/admin/app/utils/stats.js

```
@@ -135,21 +147,36 @@ export const statsStaticColors = [
     '#A568FF', '#7B7BFF', '#B3CEFF', '#D4ECF7', '#EFFDFD', '#F7F7F7'
 ];
 
-export const getCountryFlag = (countryCode) => {
+/**
+ * Converts a country code to corresponding flag emoji
+ * @param {string} countryCode - Two-letter ISO country code
+ * @returns {string} Flag emoji for the specified country code
+ */
+export function getCountryFlag(countryCode) {
     if (!countryCode || countryCode === null || countryCode.toUpperCase() === 'ᴺᵁᴸᴸ') {
         return '🏳️';
```

Biome (1.9.4)

[error] This expression always returns false, because the string is converted and will never match

This call convert the string to upper case

... but this value is not in upper case

Unsafe fix: Use upper case string value.


(lint/correctness/noStringCaseMismatch)